doc/source/configuration/magnum-capi.rst

```diff
@@ -1,45 +1,127 @@
+
+The final step before beginning deployment of the CAPI management cluster is to provide some cloud credentials. It is recommended that the CAPI management cluster is deployed in an isolated OpenStack project. After creating the target project (preferably using openstack-config), generate an application credential for the project using the Identity tab in Horizon and then download the corresponding ``clouds.yaml`` file and place it in ``environments/<site-specific-name>/clouds.yaml``.
+
+To deploy the CAPI management cluster using this site-specific environment, run
+
+.. code-block:: bash
+
+    # Activate the environment
+    ./bin/activate <site-specific-name>
+
+    # Install or update the local Ansible Python venv
+    ./bin/ensure-venv
+
+    # Install or update Ansible dependencies
+    ansible-galaxy install -f -r ./requirements.yml
+
+    # Run the provision playbook from the azimuth-ops collection
+    # NOTE: THIS COMMAND RUNS A DIFFERENT PLAYBOOK FROM
+    # THE STANDARD AZIMUTH DEPLOYMENT INSTRUCTIONS
+    ansible-playbook stackhpc.azimuth_ops.provision_capi_mgmt
+
+The general running order of the provisioning playbook is the following:
+
+- Ensure Terraform is installed locally
+
+- Use Terraform to provision the seed VM (and create any required internal networks, volumes etc.)
+
+- Install k3s on the seed (with all k3s data stored on the attached Cinder volume)
+
+- Install the required components on the k3s cluster to provision the HA cluster
+
+- Provision the HA cluster
+
+- Install the required components on the HA cluster to manage Magnum user clusters
+
+Once the seed VM has been provisioned, it can be accessed via SSH by running ``./bin/seed-ssh`` from the root of the azimuth-config repository. Within the seed VM, the k3s cluster and the HA cluster can both be accessed using the pre-installed ``kubectl`` and ``helm`` command line tools. Both of these tools will target the k3s cluster by default; however the ``kubeconfig`` file for the HA cluster can be found in the seed's home directory (named e.g. ``kubeconfig-capi-mgmt-<site-specific-name>.yaml``).
+
+*Note* - The provision playbook is responsible for copying the HA ``kubeconfig`` to this location *after* the HA cluster is up and running. If you need to access the HA cluster while it is still deploying, the ``kubeconfig`` file can be found stored as a Kubernetes secret on the k3s cluster.
```

Comment:
Same as before: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes
```suggestion
.. note::
    The provision playbook is responsible for copying the HA ``kubeconfig`` to this location *after* the HA cluster is up and running. If you need to access the HA cluster while it is still deploying, the ``kubeconfig`` file can be found stored as a Kubernetes secret on the k3s cluster.
```